src/strategies/strategy-base.sol

```diff
@@ -358,20 +362,21 @@ abstract contract StrategyBase {
 
     function _distributePerformanceFeesAndDeposit() internal {
         uint256 _want = IERC20(want).balanceOf(address(this));
+        uint256 _want_treasury = _want.mul(performanceTreasuryFee).div(performanceTreasuryMax);
+        uint256 _want_dev = _want.mul(performanceDevFee).div(performanceDevMax);
 
-        if (_want > 0) {
+
+        if (_want_treasury > 0) {
             // Treasury fees
-            IERC20(want).safeTransfer(
-                IController(controller).treasury(),
-                _want.mul(performanceTreasuryFee).div(performanceTreasuryMax)
-            );
+            IERC20(want).safeTransfer(IController(controller).treasury(), _want_treasury);
+        }
 
+        if (_want_dev > 0) {
             // Performance fee
-            IERC20(want).safeTransfer(
-                IController(controller).devfund(),
-                _want.mul(performanceDevFee).div(performanceDevMax)
-            );
+            IERC20(want).safeTransfer(IController(controller).devfund(), _want_dev);
+        }
 
+        if (_want > 0) {
```

Comment:
why did you add this `if` case?
this `_want` isn't correct because you didn't update it after sending to treasury and devfund.

Reply:
I didn't add this - deposit() was within this check previously, as it should be.

`deposit()` is an abstract function that invests the funds in the strategy (staking in a sushi LP, going to the yearn jar, staking LQTY etc) after it's harvested the earnings and paid fees. That function doesn't take an argument and gets its own balance - there's no need to adjust `_want` to the post-fee amount.

If the performance + dev fees were set to >=100% when summing then yes this _want could become zero but that degenerate case isn't likely and isn't worth optimizing the gas of IMO (correctness is unaffected since _want isn't used after this).